napari_skimage_regionprops/_load_csv.py

```diff
@@ -7,28 +7,39 @@
     import warnings
     warnings.warn(str(e))
 
-
 @register_function(menu="Measurement > Load from CSV (nsr)")
-def load_csv(csv_filename:"magicgui.types.PathLike", labels_layer: "napari.layers.Labels", viewer: "napari.Viewer" = None):
+def load_csv(csv_filename:"magicgui.types.PathLike", labels_layer: napari.layers.Labels, limit_number_visible_rows: int =None, viewer: "napari.Viewer" = None):
```

Comment:
One more question @thorstenwagner : Is there any reason why you removed the quotation marks here?

Reply:
No, my fault sorry. I changed it back.